hack/charts/cluster-api-operator/values.yaml

```
@@ -24,7 +24,6 @@ image:
 env:
   manager: []
 healthAddr: ":8081"
-metricsBindAddr: "127.0.0.1:8080"
 diagnosticsAddress: "8443"
```

```suggestion
diagnosticsAddress: ":8443"
```

One detail, the address needs to be a FQDN